commit 74cf2a6ed74471ff1d9088e8d9132fc9132861fb
Author: TJ Saunders <tj@castaglia.org>
Date:   Mon Apr 11 14:00:05 2016 -0700

    Downgrade log messages about header/library version mismatches for SQLite,
    libmemcached from WARN to INFO.

diff --git a/contrib/mod_sql_sqlite.c b/contrib/mod_sql_sqlite.c
index cd6fb53..81ee4a4 100644
--- a/contrib/mod_sql_sqlite.c
+++ b/contrib/mod_sql_sqlite.c
@@ -1059,7 +1058,7 @@ static int sql_sqlite_init(void) {
    * For now, we only log if there is a difference.
    */
   if (strcmp(sqlite3_libversion(), SQLITE_VERSION) != 0) {
-    pr_log_pri(PR_LOG_WARNING, MOD_SQL_SQLITE_VERSION
+    pr_log_pri(PR_LOG_INFO, MOD_SQL_SQLITE_VERSION
       ": compiled using SQLite version '%s' headers, but linked to "
       "SQLite version '%s' library", SQLITE_VERSION, sqlite3_libversion());
   }
diff --git a/modules/mod_memcache.c b/modules/mod_memcache.c
index e88cba8..80ab0db 100644
--- a/modules/mod_memcache.c
+++ b/modules/mod_memcache.c
@@ -364,7 +362,7 @@ static int mcache_init(void) {
 
   version = memcached_lib_version();
   if (strcmp(version, LIBMEMCACHED_VERSION_STRING) != 0) {
-    pr_log_pri(PR_LOG_WARNING, MOD_MEMCACHE_VERSION
+    pr_log_pri(PR_LOG_INFO, MOD_MEMCACHE_VERSION
       ": compiled using libmemcached-%s headers, but linked to "
       "libmemcached-%s library", LIBMEMCACHED_VERSION_STRING, version);
 
